tests/rkt_api_service_test.go

```diff
@@ -0,0 +1,274 @@
+// Copyright 2015 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"syscall"
+	"testing"
+
+	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/steveeJ/gexpect"
+	"github.com/coreos/rkt/Godeps/_workspace/src/golang.org/x/net/context"
+	"github.com/coreos/rkt/api/v1alpha"
+	"github.com/coreos/rkt/common"
+)
+
+func startAPIService(t *testing.T, ctx *rktRunCtx) *gexpect.ExpectSubprocess {
+	noGid := false
+	gid, err := common.LookupGid(common.RktGroup)
+	if err != nil {
+		t.Logf("no %q group, will run api service with root, ONLY DO THIS FOR TESTING!", common.RktGroup)
+		noGid = true
+	}
+
+	installCmd := fmt.Sprintf("%s install", ctx.cmd())
+	t.Logf("Running rkt install")
+	runRktAndCheckOutput(t, installCmd, "rkt directory structure successfully created", false)
+
+	t.Logf("Running rkt api service")
+	apisvcCmd := fmt.Sprintf("%s api-service", ctx.cmd())
+	if noGid {
+		return startRktAndCheckOutput(t, apisvcCmd, "API service running", false)
+	}
+	return startRktAsGidAndCheckOutput(t, apisvcCmd, "API service running", false, gid)
```

Comment:
Is the path `/home/alban/go/src/github.com/coreos/rkt/build-rkt-0.10.0+git/bin/rkt` accessible by `nobody`?